frontend/src/metabase/pulse/components/RecipientPicker.jsx

```diff
@@ -72,9 +73,9 @@ export default class RecipientPicker extends Component {
             valueRenderer={value => value.common_name ?? value.email}
             optionRenderer={option => (
               <div className="flex align-center">
-                <span className="text-white">
+                <Text color="white">
```

Comment:
Needed this to not be a `span` to avoid having two `span` elements for e2e tests